ui/imports/shared/views/chat/TransactionBubbleView.qml

```diff
@@ -126,8 +126,7 @@ Item {
                 id: txtError
                 color: Style.current.danger
                 visible: root.isError
-                //% "Something has gone wrong"
-                text: qsTrId("something-has-gone-wrong")
+                text: qsTr("Token not found on your current network")
```

Comment:
The error is always about token not found? Is there a scenario where something else could have gone wrong?

Reply:
Yes, it **should** be, in `getTransactionDetails`  we try to find the contract and if we cannot then we return `{}`